src/main/webapp/index.html

```diff
@@ -127,16 +128,61 @@ <h2>Metrics</h2>
                 </div>
 
                 <div id="learnMore">
-                        <h2>Want to learn more about Open Liberty and MicroProfile?</h2>
-                        <p>Continue your winning streak and keep building successful apps with the help of our guides. Our MicroProfile guides are a great place to start if you want to build something similar to this sample app.</p>
-                        <a href="https://openliberty.io/guides/?search=MicroProfile&key=tag"><button id="mpGuidesButton">Check out our MicroProfile guides</button></a>
-                    </div>
+                    <h2>Want to learn more about Open Liberty, MicroProfile, and Jakarta EE?</h2>
+                    <p>Continue your winning streak and keep building successful apps with the help of our guides. Our MicroProfile and Jakarta EE guides are a great place to start if you want to build something similar to this sample app.</p>
+                    <span>
+                        <a href="https://openliberty.io/guides/?search=MicroProfile&key=tag"><button class="checkOutGuidesButton">Check out our MicroProfile guides</button></a>
+                        <a href="https://openliberty.io/guides/?search=jakarta%20ee&key=tag"><button class="checkOutGuidesButton">Check out our Jakarta EE guides</button></a>
```

Comment:
Do these links need `target="_blank" rel="noopener"`?

Reply:
I don't think there's a need for noopener since we trust openliberty.io domain (no security concern). I'll add _blank  to them